src/autopas/selectors/tuningStrategy/MPIParallelizedStrategy.h

```diff
@@ -83,6 +84,38 @@ class MPIParallelizedStrategy : public TuningStrategyInterface {
     }
   }
 
```

Comment:
This note refers to the constructor, lines 39-51, but is placed here because I can not comment directly on the constructor.

1. The constructor does not initialize `_bucket`
2. Clang tidy suggests passing the sets by value and move-assign them just like it is already done with the tuning strategy.

Reply:
resolved by commit 15d651b9a162190058e6a25f7be018da8dd96e29